cloudfour/cloudfour.com-patterns
# Remove deprecated Sass slash division

## Overview

Sass has deprecated slashes for division: https://sass-lang.com/documentation/breaking-changes/slash-div

This PR accounts for this in two ways:

- Replaces occurrences in handwritten styles with `math.div` (most of this was done automatically using `sass-migrator`)
- Updates our aspect ratio tokens to use JS so their values will be evaluated beforehand, removing the need for division to occur in Sass for those dynamically generated files

## Testing

1. Confirm Sass warnings are no longer present in Terminal output
2. Confirm locally or via deploy preview that there are no regressions in patterns that used division

---

- Fixes #1263
